src/domains/Find/page/FindIDPage.jsx

```
@@ -19,23 +22,116 @@ const FindIDPage = () => {
   } = useForm()
   const [validationStatus, setValidationStatus] = useState(null)
   const navigate = useNavigate()
+  const [isCheckingEmail, setIsCheckingEmail] = useState(false)
+  const { findID, setFindID } = useFindIDStore()
 
+  // const onSubmit = (data) => {
+  //   if (validationStatus === 'success') {
+  //     navigate(ROUTER_PATHS.FIND_ID_COMPLETE_PAGE)
+  //   }
+  // }
   const onSubmit = (data) => {
```

ESLint

[error] 'data' is defined but never used.

(no-unused-vars)